flight/targets/brainre1/board-info/board_hw_defs.c

```diff
@@ -68,7 +71,12 @@ static const struct pios_annunc pios_annuncs[] = {
 		},
 		.remap = 0,
 		.active_high = true,
-	}
+	},
+#if defined(PIOS_INCLUDE_RE1_FPGA)
```

Comment:
It's necessary because the fpga support isn't linked into the bootupdater, so it gets a link error.